polaris-react/src/components/IndexFilters/IndexFilters.scss

```
@@ -107,6 +107,10 @@ $spinner-size: 20px;
       // stylelint-enable
     }
   }
+
+  #{$se23} & {
+    padding: var(--p-space-3);
```

Looking at spacing in design on search and sort buttons, I think this is supposed to be 8px, not 12px. 
```suggestion
    padding: var(--p-space-2);
```